docs/installation/application.rst

```diff
@@ -0,0 +1,182 @@
+************************
+Application Installation
+************************
+
+**Last Updated:** March 13, 2019
+
+Once you have the portal configured and setup with all the required services, the next step is to install Tethys applications on to your portal.
+
+1. Download App Source Code
+===========================
+
+You will need to copy the source code of the app to the server. There are many methods for accomplishing this, but one way is to create a repository for your code in GitHub. To download the source from GitHub, clone it as follows::
+
+    $ cd $TETHYS_HOME/apps/
+    $ sudo git clone https://github.com/username/tethysapp-my_first_app.git
+
+.. tip::
+
+    Substitute "username" for your GitHub username or organization and substitute "tethysapp-my_first_app" for the name of the repository with your app source code.
+
+2. Install the App
+==================
+
+Using Install (Recommended):
+-------------------------
+
+Using the ``install`` cli command, you can install the application in one quick step. The ``install`` command will install all the dependencies, link any services that you might need for your application and can also rely on the portal configuration to link default services to your application (as configured by the portal administrator).
+The install command uses three configuration files:
+
+.. _tethys_install_yml:
+
+install.yml 
+--------
+
+This file is generated with your application scaffold. Please do NOT list any dependencies in setup.py. Instead list them in the :file:`install.yml` file. This file should be committed with your application code in order to aid installation on a Tethys Portal
+
+.. code-block:: python
+
+	# This file may be committed to your app code.
+	version: 1.0
+	# This should match the app - package name in your setup.py
+	name: hydroexplorer
+
+	conda:
+	  # Putting in a skip true param will skip the entire section
+	  skip: false
+	  channels:
+	  	- tacaswell
+	  dependencies:
+	    - pyshp=2.0.0
+	    - geojson
+	    - shapely
+	  post:
+  	    - ./test.sh
+
+
+**install.yml Options:**
+
+* **version**: Indicated the version of the :file:`install.yml` file. Current default : 1.0
+* **name**: This should match the app-package name in your setup.py
+
+* **conda/skip**: If enabled, it will skip the installation of dependencies and channels
+
+* **conda/channels**: List of conda channels that need to be searched for dependency installation. Only specify any conda channels that are apart from the default. 
```

Comment:
Yes the defaults are included as specified in the user's condarc file. This will add the channels to the conda rc. Maybe that's not the best way to do it...?

Reply:
Yeah, that's not the best way to handle it. Let's not modify the condarc. Instead, run it like we do with a one off commandline call with the channels argument: i.e.: conda install -c conda-forge gdal

Reply:
I am going to need some extra clarification on this one.

Reply:
Resolved